### ISSUE TITLE
Task Management & Version Control Video v0.4

### ISSUE THREAD

**Issue by Mehrabbruno** [wizardamigos/wizardamigos.github.io#126]
## `@todo`

#124 

- [x] Create - Presentation slides for github issues top level comments for tasks v0.0.2
   - `@input` 📦 [`Presentation slides for github issues top level comments for tasks v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/By3WaDmLs) from #124 
   - [x] discussion on this presentation
   - [x] Update the presentation slides
   - `@output` 📦  [`Presentation slides for github issues top level comments for tasks v0.0.2`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/Sk6xtSSIs) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/126#issuecomment-1320355614)

---

- [x] Create - Presentation Slides for creating new issues and markdown v0.0.1
   - [x] discussion on this presentation
   - [x] create new slides and add notes for this new presentation
   - `@output` 📦 [`Presentation Slides for creating new issues and markdown`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/S1UnvNrLs) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/126#issuecomment-1320355614)

---

- [x] github issues top level comments for tasks [4/6] v0.0.2
   - `@input` 📦 [`github issues top level comments for tasks [4/6] v0.0.1`](https://youtu.be/AjFuV7iT-kQ) from #124 
   - [x] remove the markdown section from the video
   - [x] hide the @info tag from  the video
   - `@output` 📦 [`github issues top level comments for tasks [4/6] v0.0.2`](https://youtu.be/kWNwBEGYiH4) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/126#issuecomment-1320355614)

---

- [x] Create - Presentation slides for [Connecting Figma With GitHub] v0.0.2
   - `@input` 📦 [`Presentation slides for [Connecting Figma With GitHub] v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/Sy5vVafLs) from #124 
   - [x] discussion on this presentation
   - [x] Update the presentation slides
   - `@output` 📦 [`Presentation slides for [Connecting Figma With GitHub] v0.0.2`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/ByUFZ8S8s) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/126#issuecomment-1320355614)

---

- [x] Connecting Figma With GitHub [3/6] v0.0.2
   - `@input` 📦 [`Connecting Figma With GitHub [3/6] v0.0.1`](https://youtu.be/IRKltX3-8Lk) from #124 
   - [x] show how to add the github issue link to Figma 
   - `@output` 📦 [`Connecting Figma With GitHub [3/6] v0.0.2`](https://youtu.be/80SammNKzMw) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/126#issuecomment-1320355614)

---

- [x] Create - Presentation slides for [Connecting Figma With GitHub] v0.0.3
   - `@input` 📦 [`Presentation slides for [Connecting Figma With GitHub] v0.0.2`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/ByUFZ8S8s)
   - [x] Update the presentation slides
   - [x] read feedback
   - `@output` 📦 [`Presentation slides for [Connecting Figma With GitHub] v0.0.3`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/rkhQ_OZB3#/) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/126#issuecomment-1550424827)

---

- [x] Connecting Figma With GitHub [3/6] v0.0.3
   - `@input` 📦 [`Connecting Figma With GitHub [3/6] v0.0.1`](https://youtu.be/IRKltX3-8Lk) from #124 
   - [x] update the previous video with the newer slides
   - [x] update tasks and worklogs etc
   - `@output` 📦 [`Connecting Figma With GitHub [3/6] v0.0.3`](https://youtu.be/GATZ1Pc6ORg) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/126#issuecomment-1550424827)

---

- [x] creating new issues and markdown v0.0.1
   - `@input` 📦 [`Presentation Slides for creating new issues and markdown`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/S1UnvNrLs)
   - [x] record a video for creating new issues and markdown
   - `@output` 📦 [ `creating new issues and markdown v0.0.1`](https://youtu.be/y5Ne-Eze3Hc) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/126#issuecomment-1320355614)

---

- [x] Create a playlist of Task Management & Version Control Video v0.0.3
   - `@input` 📦 [`github issues top level comments for tasks [4/6] v0.0.2`](https://youtu.be/kWNwBEGYiH4)
   - `@input` 📦 [`Connecting Figma With GitHub [3/6] v0.0.2`](https://youtu.be/80SammNKzMw)
   - `@input` 📦 [ `Creating new issues and markdown v0.0.1`](https://youtu.be/y5Ne-Eze3Hc)
   - [x] create a playlist v0.0.3
   - `@output` 📦 [Task Management and version control playlist_v0.0.3](https://youtube.com/playlist?list=PLFimBx2yWz_XXdwagxRe4rW48vwlPYHkz) from [comment](https://github.com/wizardamigos/wizardamigos.github.io/issues/126#issuecomment-1556044863)

---

- [x] Create a playlist of Tasks Management & Version Control Video v0.0.4
   - `@input` 📦 [Task Management & Version Control playlist_v0.0.1](https://youtube.com/playlist?list=PLFimBx2yWz_Xr2hvFo3RJ-sFYteZAGvpV) from #124 
   - `@input` 📦 [Task Management & Version Control playlist v0.0.2](https://youtube.com/playlist?list=PLFimBx2yWz_WSTH8uoiSE4dHsvVUeXsDZ) from #124 
   - `@input` 📦 [Task Management and version control playlist_v0.0.3](https://youtube.com/playlist?list=PLFimBx2yWz_XXdwagxRe4rW48vwlPYHkz)
   - [x] [1/6] Intro to Figma vc&tm
   - [x] [2/6] Version Control on Figma v0.0.3
   - [x] [3/6] Connecting Figma with GitHub v0.0.3
   - [x] [4/6] Github issues top level comments for task v0.0.2
   - [x] [5/6] @input, @output, next and their status v0.0.1
   - [x] [6/6] worklog comment structure v0.0.1
   - [x] Creating new issues and markdown v0.0.1
   - `@output` 🏁 [Task Management and Version Control v0.0.4](https://youtube.com/playlist?list=PLFimBx2yWz_XETXchrUK6c9mA-8qSjNNY) from [comment](https://github.com/wizardamigos/wizardamigos.github.io/issues/126#issuecomment-1556048416)

**Comment by Mehrabbruno** [comment]
# Tasks

- Create - Presentation Slides for creating new issues and markdown v0.0.1
   - discussion on this presentation - `13min`
   - create new slides and add notes for this new presentation - `1h51min`
   - `@output` 📦 [`Presentation Slides for creating new issues and markdown`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/S1UnvNrLs)

---

# Feedback

I created a new video to be placed after the github top comment issue. Only talking about the markdown and creating a new issue.

---

# Worklogs

---

# Proposals

**Comment by serapath** [comment]
**feedback `2022.11.25`**


# `Introduction to Figma Version control and Task Management [1/6] v0.0.1`
:+1: perfect

---

# `github issues top level comments for tasks [4/6] v0.0.2`
:+1: perfect

---

# `Worklog comment structure [6/6] v0.0.1`
:+1: perfect

---

# `Version Control on Figma [2/6] v0.0.1`
**NEW PROBLEM - after some feedback from other people :-)

see https://youtu.be/zhudzjCAeVM?t=161

![popover component](https://imgur.com/ntl2lWe.png)

please edit the old `popovers` wireframes. They are not supposed to contain versioned content. In the content area of that version it shows all the individual component names with their individual versions, but that made no sense to people, because the popover component was a component that included all these designs inside itself **without independent design parts with versioned names** ...and was later marked as deprecated by us when we split those into individual successor components with their own page/name/version.

Multiple people were confused by it when i shared the video with them and they all asked how come that the popover component version has design content that has individual version names but is not listed in dependencies/dependents ...and i think the issue is that in the old popover component those designs were not yet independent components before we split them.


...
Could you fix that?
**We need `Version Control on Figma [2/6] v0.0.2`**


---

# `Connecting Figma With GitHub [3/6] v0.0.2`
oh no :-) haha, **almost perfect**

The link in figma to the project management software should link **to the worklog comment** (in which the output document was introduced), because the whole issue with all comments as such has potential lots of output links and many figma components would share the same issue link, so we should link instead directly to the github worklog comment.

Could you fix that part when you copy the github issue link instead of the worklog comment link?
 
**Question:**
Instead of re-recording the video, could you just record the part where you copy the comment link and use that on figma and then cut the video to insert that?
That should be less work - but if re-recording is quicker, thats fine too.

**additionaly - also to mention:**
The "publish" button needs to be pressed only once, while the github issue link is meant to be added to every component version, but a user does not need to click publish every time they make a new component - maybe that's worth mentioning.

---

# new `Create new issues and markdown text v0.0.1`

1. you mention more information about the input/output would be shared in another video, but that video doesnt seem to exist

2. the `example.com` link in both the link to the button component and the comment is a bit confusing. I think it would be good for people to see how to get a github worklog comment link and the button component repo could be a link to a figma page to make the example more realistic and meaningful

Otherwise: :+1: great video :-)


---

# `@inputs, @output, next and their status [5/6] v0.0.2`
In the video `Create new issues and markdown text v0.0.1` you mention this video,
which will share more information about the input/output status, but this video
doesn't seem to exist yet, so below i will share all feedback that i previously shared regarding the next version of this video :-)

### feedback

**The new version of this input/output/next/from/status video should mention:**
* a task with no output is a warning sign and first discussed :-)
* that tasks with outputs should always be listed before they output is used as an input
* a linked sub task doesnt have an output, because the outputs are produced "inside" the linked issue.
* every issue needs at least one output
* each issue needs at least one output
* each task needs at least one output, unless it belongs to a sub task group, in which case there needs to be at least one output at the end of the task group
* only `@output` :package: document links have a **from [`comment`](#)** link and why.
 * **BUT** it would be good to introduce things more systematically
   * so we first explain inputs/outputs and their status **and then** we explain those details :-)
   
**And:**
1. This video should explain details of creating and updating inputs/outputs in markdown
2. How to use this output as an input link in multiple other places, sub issues and different issues.
3. How to set the `next` and `from` fields correctly
4. How to set the initial status as `:planned:` and what other status exists and when they are used:
   * :question: = input/output is planned
   * :factory: = input/output is in progress (only in sub task groups)
     * should be introduced in a sub task group with an `@output`
     * where only the first task in a sub task group is done
   * :package: = input/output is done
   * :package: is initial project input **`from`** :crystal_ball:
   * :question: is final project output
     * next :checkered_flag:
   * :checkered_flag::factory: is final project output
     * next :checkered_flag:
   * :checkered_flag::package: is final project output
     * next :checkered_flag:

Of course to mention the rules:
1. an input where output was created in the same issue doesn't need a **from**
2. An `output` used in the same issue needs no **next**.
3. And every `output` needs a **from [`comment`](#)**,
which can be mentioned, but he details of the "from comment" will be explained in the next video (just shortly showing such a comment is ok)
4. An initial project input is **from** :crystal_ball:
5. A final project output has **next** :checkered_flag: 

...
Could you fix that?
**We need `@inputs, @output, next and their status [5/6] v0.0.2`**

---


## lastly - `a new playlist version` :-)

We need a new playlist that puts it all together :-)

**Comment by Mehrabbruno** [comment]
### Tasks

- Create - Presentation slides for [Connecting Figma With GitHub] v0.0.3
   - Update the presentation slides - `29min`
   - read feedback - `14min`
   - `@output` 📦 [`Presentation slides for [Connecting Figma With GitHub] v0.0.3`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/rkhQ_OZB3#/)
- Connecting Figma With GitHub [3/6] v0.0.3 - `39min`
   - update the previous video with the newer slides
   - update tasks and worklogs etc - `14min`
   - `@output` 📦 [`Connecting Figma With GitHub [3/6] v0.0.3`](https://youtu.be/GATZ1Pc6ORg)

---

### Feedback 

----

### Worklog

[worklog-148](https://youtu.be/WoPAgH9IhVw)

---

### Proposals

**Comment by serapath** [comment]
**feedback `2023.05.20`**

great. yeah, al fine.
let's finish all the tasks and link the final results, so the top level comment where we started to make this playlist for figma version control and task management can be closed.

It would be great, if you could link all issues properly and mark remaining tasks as done and create the final playlist, use all
* final video
* final slide decks
* final scripts
* etc...

as @input for that task and final worklog and then immediately mark them all as final output but with the

# :checkered_flag:
to mark list all the inputs also as final output including the playlist :-)

**Comment by Mehrabbruno** [comment]
### Tasks

- Create a playlist of Task Management & Version Control Video v0.0.3 - `39min`
   - create a playlist v0.0.3
   - change the names of all the playlist on YouTube and Github comment
   - `@output` 📦 [Task Management and version Control v0.0.3](https://youtube.com/playlist?list=PLFimBx2yWz_XXdwagxRe4rW48vwlPYHkz)

---

### Feedback 

- wrapped up videos of playlist 3

----

### Worklog

---

### Proposals

**Comment by Mehrabbruno** [comment]
### Tasks

- Create a playlist of Tasks Management & Version Control Video v0.0.4  - `11min`
   - add the following videos
      - Intro to Figma vc&tm
      - Version Control on Figma v0.0.3
      - Connecting Figma with GitHub v0.0.3
      - Github issues top level comments for task v0.0.2
      - @input, @output, next and their status v0.0.1
      - worklog comment structure v0.0.1
      - Creating new issues and markdown v0.0.1
   - `@output` 📦 [Task Management and Version Control v0.0.4](https://youtube.com/playlist?list=PLFimBx2yWz_XETXchrUK6c9mA-8qSjNNY)

---

### Feedback 

- wrapped up videos of playlist 4

----

### Worklog

---

### Proposals
